Dockerfile

```diff
@@ -5,22 +5,29 @@ FROM ${UBI_IMAGE} as ubi
 
 FROM ${GO_IMAGE} as builder
 ARG TAG="" 
+ARG K3S_ROOT_VERSION=v0.6.0-rc3
 RUN apt update     && \ 
     apt upgrade -y && \ 
     apt install -y ca-certificates git
 
-RUN git clone --depth=1 https://github.com/coreos/flannel.git /go/src/github.com/coreos/flannel
+RUN mkdir -p /tmp/xtables && \
+    curl -L https://github.com/rancher/k3s-root/releases/download/${K3S_ROOT_VERSION}/k3s-root-xtables-amd64.tar -o /tmp/xtables/k3s-root-xtables.tar && \
+    tar -C /tmp/xtables -xvf /tmp/xtables/k3s-root-xtables.tar
+
+RUN git clone --depth=1 https://github.com/rancher/flannel.git /go/src/github.com/coreos/flannel
```

Comment:
Can we clone to rancher/flannel? I think leaving the coreos ref in here might be confusing going forward.

Reply:
This would cause GOPATH issues with vendoring, from my testing

Reply:
Can you add a comment to that effect?

Reply:
Doing some further testing it appears that a simple smoke test does work in this specific `image-build` pipeline for cloing into `/go/src/github.com/rancher/flannel`, I can update to reflect this change